src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -330,8 +341,6 @@ else if (hasSuppressionLineColumnNumber) {
             }
             if (LOG.isDebugEnabled()) {
                 LOG.debug("Checkstyle debug logging enabled");
-                LOG.debug("Running Checkstyle with version: "
-                        + Main.class.getPackage().getImplementationVersion());
```

Comment:
Not working without jar and no real benefits to have it not possible to cover in pitest.
We prefer pitest over this debug log.